etna/lib/etna/metrics.rb

```diff
@@ -0,0 +1,26 @@
+
+module Etna
+  class MetricsExporter
+    def initialize(app, path: '/metrics')
```

Comment:
Middleware for any etna app to export metrics.  After installing this to config.ru, you just need to add the app as a target in the `prometheus.yml` file to have its metrics collected.